src/VisualStudio/Core/Def/Implementation/TableDataSource/AbstractTableDataSource.cs

```diff
@@ -24,16 +24,11 @@ public AbstractTableDataSource()
             Subscriptions = ImmutableArray<SubscriptionWithoutLock>.Empty;
         }
 
-        public virtual void OnProjectDependencyChanged(Solution solution)
```

Comment:
dependency tracking has moved to error list itself. removed our project dependency tracking code.